beta/src/pages/learn/importing-and-exporting-components.md

```diff
@@ -4,24 +4,22 @@ title: Importing and Exporting Components
 
 <Intro>
 
-The magic of components lies in their reusability: you can create components that are composed of other components. But as you nest more and more components, it often makes sense to start splitting them into different files. This lets you keep your files easy to scan and reuse components in more places.
-
+הקסם של קומפוננטות טמון בשימוש החוזר שלהם: ניתן ליצור קומפנטטות המורכבות מקומפוננטות אחרות. אבל ככל שאתם מתחילים לקנן יותר ויותר קומפוננטות, לעתים קרובות הגיוני יותר להתחיל לפצל אותם לקבצים שונים. זה יאפשר לכם לשמור על הקבצים שלכם קריאים ולעשות שימוש חוזר בקומפוננטות במקומות נוספים.
```

Comment:
קומפנטטות typo

Reply:
how was this done in other places in the docs?

Reply:
In the previous docs we used לקנן, your PR is the first one that addresses that word on the new beta docs
I think the best translation for that sentence is 
אבל ככל שאתם מתחילים להשתמש ביותר ויותר קומפוננטות בתוך קומפוננטות אחרות,

Reply:
i would translate "reusability" as - יכלת לשימוש חוזר